test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -68,3 +70,350 @@ def forward(self, x):
+                        ns.call_function(torch.ops.quantized_decomposed.dequantize_per_tensor.default),
+                        ns.call_function(torch.ops.aten.convolution.default),
+                        ns.call_function(torch.ops.aten.relu_.default if inplace_relu else torch.ops.aten.relu.default),
+                        ns.call_function(torch.ops.quantized_decomposed.quantize_per_tensor.default),
+                        ns.call_function(torch.ops.quantized_decomposed.dequantize_per_tensor.default),
+                    ]
+                    self.checkGraphModuleNodes(m,
+                                               expected_node_occurrence=node_occurrence,
+                                               expected_node_list=node_list)
+
+    @skipIfNoX86
+    def test_conv2d_binary_with_quantizer_api(self):
+        class Conv2DType(Enum):
+            left = 1
+            right = 2
+            both = 3
+
+        class Mod(torch.nn.Module):
+            def __init__(self,
+                         inplace_add: bool = False,
+                         conv2d_type: Conv2DType = Conv2DType.left,
+                         use_bias: bool = False,
+                         ) -> None:
+                super().__init__()
+                self.conv = torch.nn.Conv2d(
+                    in_channels=3, out_channels=3, kernel_size=3, stride=1, padding=1, bias=use_bias
+                )
+                self.conv2 = torch.nn.Conv2d(
+                    in_channels=3, out_channels=3, kernel_size=3, stride=1, padding=1, bias=use_bias
+                )
+                self.relu = nn.ReLU()
+                self.inplace_add = inplace_add
+                self.conv2d_type = conv2d_type
+
+            def forward(self, x):
+                if self.conv2d_type == Conv2DType.left:
+                    if self.inplace_add:
+                        tmp = self.conv(x)
+                        tmp += self.relu(x)
+                        return tmp
+                    else:
+                        return self.conv(x) + self.relu(x)
+                elif self.conv2d_type == Conv2DType.right:
+                    if self.inplace_add:
+                        tmp = self.relu(x)
+                        tmp += self.conv(x)
+                        return tmp
+                    else:
+                        return self.relu(x) + self.conv(x)
+                elif self.conv2d_type == Conv2DType.both:
+                    if self.inplace_add:
+                        tmp = self.conv(x)
+                        tmp += self.conv2(x)
+                        return tmp
+                    else:
+                        return self.conv(x) + self.conv2(x)
+
+
+        inplace_add_list = [True, False]
+        conv2d_type_list = [Conv2DType.left, Conv2DType.right, Conv2DType.both]
+        use_bias_list = [True, False]
+
+        with override_quantized_engine("x86"):
+            with torch.no_grad():
```

Comment:
can you use two `with` in the same statement to reduce indentation?

Reply:
Thanks and changed.